parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -306,33 +309,35 @@ private WriteBuilder createContextFunc(
       return this;
     }
 
+    // Utility method to get the column path
+    private String getParquetColumnPath(
+        Map<Integer, String> fieldIdToParquetPath, String colPath, String configStr) {
+      Types.NestedField fieldId = schema.findField(colPath);
```

Comment:
nit: should it be called 'field' instead of a fieldId?

Reply:
Removed getParquetColumnPath in the latest patch.